topdown/tokens_test.go

```diff
@@ -113,7 +113,7 @@ OHoCIHmNX37JOqTcTzGn2u9+c8NlnvZ0uDvsd1BmKPaUmjmm
 	})
 	t.Run("Time", func(t *testing.T) {
 		now := time.Now()
-		wallclock := ast.IntNumberTerm(int(now.UnixNano()))
+		wallclock := ast.UIntNumberTerm(uint64(now.UnixNano()))
```

Comment:
At the top of the tests, we use this:
https://github.com/open-policy-agent/opa/blob/9f4c44bf7400239404d6821d15542a2a2cadf89b/topdown/tokens_test.go#L24

I suppose we could make this consistent, one way or another. What do you think? 😃

Reply:
:+1: 

I've just checked this `int64ToJSONNumber` and both methods do actually the same thing, mine having an additional conversion to `uint64`, so I guess it's better to use `ast.NumberTerm(int64ToJSONNumber(...))`, for consistency.